open_rarity/scoring/handlers/information_content_scoring_handler.py

```diff
@@ -46,6 +46,12 @@ def score_token(
     ) -> float:
         """See ScoringHandler interface.
 
+        Parameters
```

Comment:
Can we remove `normalized` completely from here? i think it's a no-op

Reply:
Ok i'm removing from base-level handler interface since it may not be applicable to types of handlers. Put it as init configs in handlers.